features/importers/base.py

```diff
@@ -0,0 +1,22 @@
+from features.models import SourceType
+
+
+class FeatureImporterBase:
```

Comment:
Optional: Could consider making this an abstract base class and decorate the properties required for each implementation with `@property @abstractmethod` so an implementer will more easily spot bugs?

Reply:
The importer API might still change quite a bit, but sure we can try and see how using an abstract class in this case feels. :+1: Currently the only "hard requirement" is the `import_features` since that's the method which gets called from the management command.